packages/core/src/Stores/contract-trade-store.js

```diff
@@ -397,7 +405,7 @@ export default class ContractTradeStore extends BaseStore {
 
     get last_contract() {
         const applicable_contracts = this.applicable_contracts();
-        const length = applicable_contracts.length;
+        const length = this.contracts[0]?.contract_info.current_spot_time ? applicable_contracts.length : -1;
```

Comment:
incorporated the change from https://github.com/binary-com/deriv-app/pull/10398 because it's necessary for my changes to work